packages/block-library/src/nextpage/editor.scss

```diff
@@ -1,4 +1,4 @@
-.editor-visual-editor__block[data-type="core/nextpage"] {
+.block-editor-block-list__block[data-type="core/nextpage"] {
```

Comment:
Noting that this was previously very broken, `.editor-visual-editor__block` doesn't exist anywhere before or after this set of changes. It should be corrected here now.